ibtk/src/lagrangian/LEInteractor.cpp

```diff
@@ -1573,8 +1573,7 @@ string_to_kernel(const std::string& kernel_fcn)
 double (*LEInteractor::s_kernel_fcn)(double r) = &ib4_kernel_fcn;
 int LEInteractor::s_kernel_fcn_stencil_size = 4;
 
-void
-LEInteractor::setFromDatabase(Pointer<Database> /*db*/)
+void LEInteractor::setFromDatabase(Pointer<Database> /*db*/)
```

Comment:
Don't we usually put return types on their own line? What did clang-format do here?

Reply:
We do. I had to check and the current `.clang-format` file is set up so that is true for all returned objects - however it is not for the 'typeless type' of void. In other files `void` is not on its own line.